SConstruct

```diff
@@ -207,6 +207,13 @@ opts.Add(BoolVariable("modules_enabled_by_default", "If no, disable all modules
 opts.Add(BoolVariable("no_editor_splash", "Don't use the custom splash screen for the editor", True))
 opts.Add("system_certs_path", "Use this path as SSL certificates default for editor (for package maintainers)", "")
 opts.Add(BoolVariable("use_precise_math_checks", "Math checks use very precise epsilon (debug option)", False))
+opts.Add(
+    BoolVariable(
+        "tinyexr_export_templates",
```

Comment:
Would it work to move this option to the tinyexr module's `config.py`, so we don't bloat the main script? I think modules support adding options, see the Mono module.

Reply:
I've tried something like this, but it makes the option no longer listed in `scons -h` and generally involves a fair amount of copypasting from the main SCsub.

```diff
diff --git a/SConstruct b/SConstruct
index fecf89dbf8..2763f3f578 100644
--- a/SConstruct
+++ b/SConstruct
@@ -208,13 +208,6 @@ opts.Add(BoolVariable("modules_enabled_by_default", "If no, disable all modules
 opts.Add(BoolVariable("no_editor_splash", "Don't use the custom splash screen for the editor", True))
 opts.Add("system_certs_path", "Use this path as SSL certificates default for editor (for package maintainers)", "")
 opts.Add(BoolVariable("use_precise_math_checks", "Math checks use very precise epsilon (debug option)", False))
-opts.Add(
-    BoolVariable(
-        "tinyexr_export_templates",
-        "Enable saving and loading OpenEXR images in export template builds (increases binary size)",
-        False,
-    )
-)
 
 # Thirdparty libraries
 opts.Add(BoolVariable("builtin_certs", "Use the built-in SSL certificates bundles", True))
diff --git a/modules/tinyexr/SCsub b/modules/tinyexr/SCsub
index bf9242cc16..d569ed92b6 100644
--- a/modules/tinyexr/SCsub
+++ b/modules/tinyexr/SCsub
@@ -5,6 +5,24 @@ Import("env_modules")
 
 env_tinyexr = env_modules.Clone()
 
+customs = ["../../custom.py"]
+profile = ARGUMENTS.get("profile", "")
+if profile:
+    if os.path.isfile(profile):
+        customs.append(profile)
+    elif os.path.isfile(profile + ".py"):
+        customs.append(profile + ".py")
+
+opts = Variables(customs, ARGUMENTS)
+opts.Add(
+    BoolVariable(
+        "tinyexr_export_templates",
+        "Enable saving and loading OpenEXR images in export template builds (increases binary size)",
+        False,
+    )
+)
+opts.Update(env_tinyexr)
+
 # Thirdparty source files
 
 thirdparty_obj = []

```

Reply:
I've pushed a second commit where I attempted this, but it fails when using `target=template_release`:

```
KeyError: 'tinyexr_export_templates':
  File "/home/hugo/Documents/Git/godotengine/godot/SConstruct", line 794:
    if config.can_build(env, selected_platform):
  File "/home/hugo/Documents/Git/godotengine/godot/modules/tinyexr/config.py", line 2:
    return env.editor_build or env["tinyexr_export_templates"]
  File "/home/hugo/.local/opt/pyston/lib/python3.8-pyston2.3/site-packages/SCons/Environment.py", line 387:
    return self._dict[key]
```

The option *is* listed in `scons -h`'s output still.